database/migrations/2021_09_28_201922_add_declaration_bio_meat_gift.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Support\Facades\DB;
+
+class AddDeclarationBioMeatGift extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('declarations', function (Blueprint $table) {
+            $table->enum('declaration_type', ['pay', 'gift', 'pay-biomeat'])->default('pay');
```

Comment:
Deze dan ook aanpassen

Reply:
nvt, het systeem is geschreven vanuit de penningmeester, dat maakt dat het een betaling is naar het biovlees potje.